Automattic/woocommerce-payments
# refactor: SettingsManager gateway icons

Fixes #

#### Changes proposed in this Pull Request

Based off of https://github.com/Automattic/woocommerce-payments/pull/1709










Slight refactor on the SVG icons displayed throughout the settings.
This allows to add icons to the "Digital wallets" section: https://d.pr/i/30dl5r
It just changes how the other icons are being displayed.
I also ensured that the icons in the "Add payment method" modal are using the shared SVGs.

I decided to move the SVGs to a React component, because it allows for more flexibility when passing additional props (like `width`, `height`, etc.)

Tested in Chrome, FF, Safari.

#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Visit http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- There should be no major visual difference compared to what is in the `develop` branch (besides the icons added to the "Digital wallets" section

-------------------
